src/app/modules/main/chat_section/controller.nim

```diff
@@ -230,4 +230,34 @@ method rejectContactRequest*(self: Controller, publicKey: string) =
   self.contactService.rejectContactRequest(publicKey)
 
 method blockContact*(self: Controller, publicKey: string) =
-  self.contactService.blockContact(publicKey)
\ No newline at end of file
+  self.contactService.blockContact(publicKey)
+
+proc convertPubKeysToJson(self: Controller, pubKeys: string): seq[string] =
```

Comment:
@borismelnik this proc should be part of `module`, not here. That's already added in previous issue, [here](https://github.com/status-im/status-desktop/pull/4270#discussion_r774975234).